algotaurus.py

```diff
@@ -788,4 +836,10 @@ def execute_code(self):
 algotaurus
     run in graphical user interface mode'''
     else:  # Run GUI version
-        root = AlgoTaurusGui()
+        code, exit_flag = '', False
+        maze_rows, maze_columns = 30, 30        
+        while not exit_flag:
+            read_cfg()
+            root = AlgoTaurusGui(code=code, maze_rows=maze_rows, maze_columns=maze_columns)
+            maze_rows, maze_columns = root.maze_rows, root.maze_columns
+            code, exit_flag = root.code, root.exit_flag
```

Comment:
We constructed many functions destroying the mainloop (language changing and resize commands). The exit_flag comes True only if the program calls the exit_command.
With this modification the program applies the changes, reconfigures itself and restarts when we change the language of the GUI or the size of the maze.